Source/Csla.Blazor/ViewModel.cs

```diff
@@ -194,12 +332,31 @@ public T Model
       }
     }
 
-    private readonly Dictionary<string, object> _info = new Dictionary<string, object>();
+    /// <summary>
+    /// Gets or sets a value indicating whether the
+    /// view model should manage the lifetime of
+    /// the business object, including using n-level
+    /// undo.
+    /// </summary>
+    //[Browsable(false)]    // TODO: ask if this is desired
```

Comment:
That attribute controls whether the property appears in things like the IDE and intellisense. I think it probably _should_ be browsable, which is the default.

Reply:
Ok. I removed the comment line.